docs/tutorials/standalone/Soil/evaporation_gilat_loess.jl

```diff
@@ -0,0 +1,356 @@
+
+# The same experiment is carried out 3 times
+# 1. No evaporation (zero flux boundary conditions)
+# 2. With evaporation but no drainage (Ksat = 0)
+# 3. With evaporation and drainage
+
+using CairoMakie
+import SciMLBase
+import ClimaTimeSteppers as CTS
+using Thermodynamics
+
+using ClimaCore
+import ClimaParams as CP
+using SurfaceFluxes
+using StaticArrays
+using Dates
+using ArtifactWrappers
+using DelimitedFiles: readdlm
+
+using ClimaLand
+using ClimaLand.Domains: Column
+using ClimaLand.Soil
+import ClimaLand
+import ClimaLand.Parameters as LP
+import SurfaceFluxes.Parameters as SFP
+
+FT = Float64;
+earth_param_set = LP.LandParameters(FT)
+thermo_params = LP.thermodynamic_parameters(earth_param_set);
+
+# Parameters
+K_sat = FT(0.01 / 3600 / 24)
+vg_n = FT(1.45)
+vg_α = FT(1.5)
+hcm = vanGenuchten{FT}(; α = vg_α, n = vg_n)
+ν = FT(0.4)
+θ_r = FT(0.04)
+S_s = FT(1e-3)
+ν_ss_om = FT(0.0)
+ν_ss_quartz = FT(0.3)
+ν_ss_gravel = FT(0.0)
+emissivity = FT(1.0)
+PAR_albedo = FT(0.2)
+NIR_albedo = FT(0.4)
+z_0m = FT(1e-3)
+z_0b = FT(1e-4)
+d_ds = FT(0.01)# 10mm
+params = ClimaLand.Soil.EnergyHydrologyParameters(
+    FT;
+    ν,
+    ν_ss_om,
+    ν_ss_quartz,
+    ν_ss_gravel,
+    hydrology_cm = hcm,
+    K_sat,
+    S_s,
+    θ_r,
+    PAR_albedo,
+    NIR_albedo,
+    emissivity,
+    z_0m,
+    z_0b,
+    earth_param_set,
+    d_ds,
+);
+
+# Boundary conditions
+# No evaporation
+zero_water_flux = WaterFluxBC((p, t) -> 0)
+zero_heat_flux = HeatFluxBC((p, t) -> 0)
+no_flux_boundary_fluxes = (;
+    top = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+    bottom = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+);
+# With evaporation
+ref_time = DateTime(2005)
+SW_d = (t) -> 0
+LW_d = (t) -> 294.15^4 * 5.67e-8
+radiation = PrescribedRadiativeFluxes(
+    FT,
+    TimeVaryingInput(SW_d),
+    TimeVaryingInput(LW_d),
+    ref_time,
+)
+# Atmos
+T_air = FT(301.15)
+rh = FT(0.38)
+esat = Thermodynamics.saturation_vapor_pressure(
+    thermo_params,
+    T_air,
+    Thermodynamics.Liquid(),
+)
+e = rh * esat
+q = FT(0.622 * e / (101325 - 0.378 * e))
+precip = (t) -> 0.0
+T_atmos = (t) -> T_air
+u_atmos = (t) -> 1.0
+q_atmos = (t) -> q
+h_atmos = FT(0.1)
+P_atmos = (t) -> 101325
+gustiness = FT(1e-2)
+atmos = PrescribedAtmosphere(
+    TimeVaryingInput(precip),
+    TimeVaryingInput(precip),
+    TimeVaryingInput(T_atmos),
+    TimeVaryingInput(u_atmos),
+    TimeVaryingInput(q_atmos),
+    TimeVaryingInput(P_atmos),
+    ref_time,
+    h_atmos;
+    gustiness = gustiness,
+)
+top_bc = ClimaLand.Soil.AtmosDrivenFluxBC(atmos, radiation)
+evap_boundary_fluxes = (;
+    top = top_bc,
+    bottom = WaterHeatBC(; water = zero_water_flux, heat = zero_heat_flux),
+)
+
+# Without drainage - wet boundary at bottom
+bottom_water_bc = MoistureStateBC((p, t) -> 0.35)
+no_drainage_boundary_fluxes = (;
+    top = top_bc,
+    bottom = WaterHeatBC(; water = bottom_water_bc, heat = zero_heat_flux),
+)
```

Comment:
I think it would be more clear/readable to define each boundary flux in the case it's used for, but this is personal preference :)

Reply:
good idea! I changed it